code/src/App.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react'
+import { BrowserRouter, Switch, Route, Redirect} from 'react-router-dom'
+import { PopularList } from 'components/PopularList'
```

Comment:
I actually learnt from my last code review that the outer dependencies (those located in node_modules) should be separated from local dependencies (all local files that we created like: components, functions, data such as arrays) with one empty space above/under each other. See the example under! 😁

import React from 'react'
import { BrowserRouter, Switch, Route, Redirect} from 'react-router-dom'

import { PopularList } from 'components/PopularList'
import { Detail } from 'components/Detail'
import { NotFound } from 'components/NotFound'